pkg/sql/codegen/pai/codegen.go

```diff
@@ -49,11 +49,11 @@ func getTableFromSelect(dataSource, trainSelect string) (string, string, error)
 		database = tableParts[0]
 		tableName = tableParts[1]
 	} else {
-		parts := strings.Split(dataSource, "://")
```

Comment:
Can call `sql.SplitDataSource`: 

https://github.com/sql-machine-learning/sqlflow/blob/6344075f4edc0bfb9f413e6c87f67483c47274cb/pkg/sql/database.go#L67-L76

Reply:
importing `pkg/sql` will cause:
```
import cycle not allowed
package sqlflow.org/sqlflow/cmd/sqlflowserver
	imports sqlflow.org/sqlflow/pkg/server
	imports sqlflow.org/sqlflow/pkg/sql
	imports sqlflow.org/sqlflow/pkg/sql/codegen/pai
	imports sqlflow.org/sqlflow/pkg/sql
```

so just keep it here.